packages/combobox/test/combobox-a11y.test.ts

```diff
@@ -0,0 +1,309 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { elementUpdated, expect, html, oneEvent } from '@open-wc/testing';
+
+import '@spectrum-web-components/combobox/sp-combobox.js';
+import '@spectrum-web-components/combobox/sp-combobox-item.js';
+import { Combobox, ComboboxOption } from '..';
+import { arrowDownEvent, fixture } from '../../../test/testing-helpers.js';
+import { a11ySnapshot, findAccessibilityNode } from '@web/test-runner-commands';
+import { TestableCombobox } from './index.js';
+import { withFieldLabel, withHelpText } from '../stories/combobox.stories.js';
+import { MenuItem } from '@spectrum-web-components/menu';
+
+const isWebKit =
+    /AppleWebKit/.test(window.navigator.userAgent) &&
+    !/Chrome/.test(window.navigator.userAgent);
+
+const comboboxFixture = async (): Promise<TestableCombobox> => {
```

Comment:
Is one of the storybook stories sufficient to test against so that we don't need another instance of this?

Reply:
I liked that the test fixture only had 4 options, but I guess we could configure the stories to accept the `options` as an argument in the stories and just feed that into the test. Idk.

Reply:
Maybe we make the story template take options as an argument?

This also links up with https://github.com/adobe/spectrum-web-components/pull/3953#discussion_r1466918896 and can be addressed later, though we should keep notes on any tasks of which we allow a deferral.